e2e/service_test.go

```diff
@@ -0,0 +1,75 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"io/ioutil"
+	"log"
+	"testing"
+
+	"github.com/mesg-foundation/engine/hash"
+	pb "github.com/mesg-foundation/engine/protobuf/api"
+	"github.com/stretchr/testify/require"
+	"google.golang.org/grpc/metadata"
+)
+
+var (
+	testServiceHash  hash.Hash
+	testInstanceHash hash.Hash
```

Comment:
```suggestion
//	testInstanceHash hash.Hash
```

this break the test thus no e2e test are run on CI